test/gurka/gurka.h

```diff
@@ -523,12 +547,12 @@ std::tuple<const baldr::GraphId,
            const baldr::DirectedEdge*,
            const baldr::GraphId,
            const baldr::DirectedEdge*>
-findEdge(const std::unique_ptr<valhalla::baldr::GraphReader>& reader,
+findEdge(valhalla::baldr::GraphReader& reader,
```

Comment:
i made the same changes in my branch but there is one important one missing here..